Error updating the NVD Data

**Environment:**
```
OS Ubuntu-22.04 LTS
Dependency-Check Core version 12.1.0
```
**Errror:**
```
dependency-check.sh --disableRetireJS --scan workspace/project/sample/ --format HTML --format JSON --format XML --out /tmp/
[INFO] Checking for updates
[WARN] An NVD API Key was not provided - it is highly recommended to use an NVD API key as the update can take a VERY long time without an API Key
[WARN] NVD API request failures are occurring; retrying request for the 15th time
[WARN] NVD API request failures are occurring; retrying request for the 16th time
[WARN] NVD API request failures are occurring; retrying request for the 17th time
[WARN] NVD API request failures are occurring; retrying request for the 18th time
[WARN] NVD API request failures are occurring; retrying request for the 19th time
[WARN] NVD API request failures are occurring; retrying request for the 20th time
[WARN] NVD API request failures are occurring; retrying request for the 21st time
[WARN] NVD API request failures are occurring; retrying request for the 22nd time
[WARN] NVD API request failures are occurring; retrying request for the 23rd time
[WARN] NVD API request failures are occurring; retrying request for the 24th time
[WARN] NVD API request failures are occurring; retrying request for the 25th time
[WARN] NVD API request failures are occurring; retrying request for the 26th time
[WARN] NVD API request failures are occurring; retrying request for the 27th time
[WARN] NVD API request failures are occurring; retrying request for the 28th time
[WARN] NVD API request failures are occurring; retrying request for the 29th time
[WARN] NVD API request failures are occurring; retrying request for the 30th time
[WARN] NVD API request failures are occurring; retrying request for the 31st time
[ERROR] Error updating the NVD Data
org.owasp.dependencycheck.data.update.exception.UpdateException: Error updating the NVD Data
        at org.owasp.dependencycheck.data.update.NvdApiDataSource.processApi(NvdApiDataSource.java:399)
        at org.owasp.dependencycheck.data.update.NvdApiDataSource.update(NvdApiDataSource.java:117)
        at org.owasp.dependencycheck.Engine.doUpdates(Engine.java:903)
        at org.owasp.dependencycheck.Engine.initializeAndUpdateDatabase(Engine.java:708)
        at org.owasp.dependencycheck.Engine.analyzeDependencies(Engine.java:634)
        at org.owasp.dependencycheck.App.runScan(App.java:269)
        at org.owasp.dependencycheck.App.run(App.java:201)
        at org.owasp.dependencycheck.App.main(App.java:93)
Caused by: io.github.jeremylong.openvulnerability.client.nvd.NvdApiException: NVD Returned Status Code: 503
        at io.github.jeremylong.openvulnerability.client.nvd.NvdCveClient._next(NvdCveClient.java:438)
        at io.github.jeremylong.openvulnerability.client.nvd.NvdCveClient.next(NvdCveClient.java:357)
        at org.owasp.dependencycheck.data.update.NvdApiDataSource.processApi(NvdApiDataSource.java:355)
        ... 7 common frames omitted
[INFO] Updating CISA Known Exploited Vulnerability list: https://www.cisa.gov/sites/default/files/feeds/known_exploited_vulnerabilities.json
[INFO] Begin database defrag
[INFO] End database defrag (293 ms)
[WARN] Unable to update 1 or more Cached Web DataSource, using local data instead. Results may not include recent vulnerabilities.
[ERROR] Unable to continue dependency-check analysis.
[ERROR] One or more fatal errors occurred
[ERROR] Error updating the NVD Data
[ERROR] No documents exist
```

What could the issue?

--- a/ant/src/site/markdown/index.md.vm
+++ b/ant/src/site/markdown/index.md.vm
@@ -35,7 +35,10 @@ Installation
 
 It is important to understand that the first time this task is executed it may
 take 10 minutes or more as it downloads and processes the data from the National
-Vulnerability Database (NVD) hosted by NIST: https://nvd.nist.gov
+Vulnerability Database (NVD) hosted by NIST: https://nvd.nist.gov.
 
 After the first batch download, as long as the task is executed at least once every
 seven days the update will only take a few seconds.
+
+The Dependency-Check team strongly recommends to [mirror the NVD database](../data/mirrornvd.html) for any operational
+integration. If not done, any service disruption of the NVD database will make the usage of Dependency-Check difficult.

--- a/maven/src/site/markdown/index.md.vm
+++ b/maven/src/site/markdown/index.md.vm
@@ -5,10 +5,13 @@ plug-in or as part of the site plug-in. The plug-in requires Maven 3.6.3 or high
 
 It is important to understand that the first time this task is executed it may
 take 20 minutes or more as it downloads and processes the data from the National
-Vulnerability Database (NVD) hosted by NIST: https://nvd.nist.gov
+Vulnerability Database (NVD) hosted by NIST: https://nvd.nist.gov.
 
 After the first batch download, as long as the plug-in is executed at least once every
 seven days the update will only take a few seconds.
+
+The Dependency-Check team strongly recommends to [mirror the NVD database](../data/mirrornvd.html) for any operational
+integration. If not done, any service disruption of the NVD database will make the usage of Dependency-Check difficult.
 
 ### Default Phase
 The dependency-check plugin is, by default, tied to the `verify` or `site` phase

--- a/src/site/markdown/data/mirrornvd.md
+++ b/src/site/markdown/data/mirrornvd.md
@@ -7,7 +7,7 @@ The NVD API and the Retire JS repository.
 Creating an offline cache for the NVD API
 ------------------------------------------------------------
 
-The Open Vulnerability Project's [vuln CLI](https://github.com/dependency-check/Open-Vulnerability-Project/tree/main/vulnz#caching-the-nvd-cve-data)
+The Open Vulnerability Project's [vuln CLI](https://github.com/jeremylong/open-vulnerability-cli/blob/main/README.md)
 can be used to create an offline copy of the data obtained from the NVD API.
 Then configure dependency-check to use the NVD Datafeed URL.
 

--- a/src/site/markdown/dependency-check-gradle/index.md.vm
+++ b/src/site/markdown/dependency-check-gradle/index.md.vm
@@ -5,10 +5,13 @@ libraries; creating a report of known vulnerable components that are included in
 
 It is important to understand that the first time this task is executed it may
 take 5-20 minutes as it downloads and processes the data from the National
-Vulnerability Database (NVD) hosted by NIST: https://nvd.nist.gov
+Vulnerability Database (NVD) hosted by NIST: https://nvd.nist.gov.
 
 After the first batch download, as long as the plugin is executed at least once every
 seven days the update will only take a few seconds.
+
+The Dependency-Check team strongly recommends to [mirror the NVD database](../data/mirrornvd.html) for any operational
+integration. If not done, any service disruption of the NVD database will make the usage of Dependency-Check difficult.
 
 #set( $H = '#' )
 

